TobiiTest/MicrosoftTranslate.cs

```diff
@@ -0,0 +1,53 @@
+using System;
+using System.Net.Http;
+using System.Text;
+// NOTE: Install the Newtonsoft.Json NuGet package.
+using Newtonsoft.Json;
+
+namespace TobiiTest
+{
+    class MicrosoftTranslator
+    {
+        static string host = "https://api.cognitive.microsofttranslator.com";
+        static string path = "/translate?api-version=3.0";
+        // Translate to German and Italian.
+        static string params_ = "&to=de&to=it";
+
+        static string uri = host + path + params_;
+
+        // NOTE: Replace this example key with a valid subscription key.
+        static string key = "0dcb155e46094a3a8be0ba81710e4f06";
+
+        static string text = "Hello world!";
+
+        async static void Translate()
```

Comment:
1) Change this method to return translated string and make the text-to-be-translated a parameter
```suggestion
        async static string Translate(string text)
```
2) Make this non-static method - similar to GoogleTranslate. The translator class should have source and target level properties and a constructor to set them.

Reply:
Okay, return type should be Task<string>